electron/src/services/electronStorage.service.ts

```diff
@@ -46,10 +46,12 @@ export class ElectronStorageService implements StorageService {
     }
 
     save(key: string, obj: any): Promise<any> {
-        if (obj instanceof Set) {
-            obj = Array.from(obj);
+        if (obj != null) {
+            if (obj instanceof Set) {
+                obj = Array.from(obj);
+            }
+            this.store.set(key, obj);
```

Comment:
I kind of prefers to use "guard clauses" for these scenarios.

```suggestion
        if (obj == null) {
            return Promise.resolve();
        }
        if (obj instanceof Set) {
        	obj = Array.from(obj);
        }
        this.store.set(key, obj);
```